nodejs/awsx/apigateway/api.ts

```diff
@@ -662,13 +752,20 @@ function addStaticRouteToSwaggerSpec(
         // Take whatever path the client wants to host this folder at, and add the
         // greedy matching predicate to the end.
         const proxyPath = directoryServerPath + "{proxy+}";
-        addSwaggerOperation(swagger, proxyPath, swaggerMethod("ANY"), createSwaggerOperationForObjectKey(directoryKey, role, "proxy"));
+        let swaggerOperation = createSwaggerOperationForObjectKey(directoryKey, role, "proxy");
```

Comment:
const.  (i'm actually surpirsed the linter isn't complaining).

Reply:
It actually is complaining -- and I'm confused about how this works since we edit it on like 756. Would love to chat w/ you about this.